manage_user_page.php

```diff
@@ -185,6 +190,17 @@
 $t_where_params = array();
 if( $f_filter === 'ALL' ) {
 	$t_where = '(1 = 1)';
+} else if( $f_findname<>"" ) {
+	$c_prefix = strtoupper(db_prepare_string($f_filter));
+	$pos = strpos($c_prefix, '-');
+	if ($pos){
+		$exclude = trim(substr($c_prefix,$pos+1));
+		$c_prefix= trim(substr($c_prefix, 0,$pos-1));
+		$t_where = "(upper(realname) like '%$c_prefix%' and upper(realname) not like '%$exclude%')";
```

Comment:
There is a function db_helper_like that provides case-insensitive like phrase based on the current db type (e.g. works with MySQL and PostgreSQL, see https://github.com/mantisbt/mantisbt/blob/release-2.16.0/core/database_api.php#L726
I expect a better performance on installations with lot's of users as I assume that your implementation leads to a full table scan.